docs/content/TextInput.mdx

```diff
@@ -124,6 +124,7 @@ render(<WithIconAndLoadingIndicator />)
         }}
         icon={XIcon}
         aria-label="Clear input"
+        tooltip-direction="nw"
```

Comment:
Could you please add another example with the `tooltip-direction` passed as a prop rather than editing the Trailing Action example here?

Reply:
I'll do it.